UWP/App.cpp

```diff
@@ -14,8 +14,11 @@
 
 #include <ppltasks.h>
 
-using namespace UWP;
+#include <UWPHelpers/LaunchItem.h>
```

Comment:
Please use `""` around files that are local to the project, and `<>` around system files.  This is our file.

-[Unknown]